sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/DetectedLanguage.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The DetectedLanguage model.
+ */
+@Fluent
+public final class DetectedLanguage {
```

Comment:
I don't think this should be a fluent class. It's a result returned from the service. It should be immutable.
https://www.baeldung.com/jackson-deserialize-immutable-objects

Reply:
update this class to immutable. We regenerated all code-gen classes with a newer swagger. FYI, currently, all codegen models are in the implementation folder. We do one-to-one transfer in the async client.